templates/semantic-ui/zenodo_rdm/macros/communities_list.html

```diff
@@ -0,0 +1,38 @@
+{#
+# This file is part of Zenodo.
+# Copyright (C) 2024 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+-#}
+
+{% macro communities_list(communities=None) %}
+  {% for community in communities %}
+    {% set community_title = community.metadata.get("title", "No title") if community.metadata %}
+    <span class="comma_separated">
+      <a
+        class="community-link"
+        href="/communities/{{ community.slug }}"
+        title="{{ community_title }}: {{_('Search')}}"
```

Comment:
just for my education: what does the title attr do? I never used it with a tag like this

Reply:
Little gray tooltip like this:
![Screenshot 2024-02-16 at 09 22 21](https://github.com/zenodo/zenodo-rdm/assets/61321254/723319cc-54a4-4134-87ff-432d2459b635)